docs/book/src/reference/creating-events.md

```diff
@@ -0,0 +1,109 @@
+# Creating Events
+
+It is often useful to publish *Event* objects from the controller Reconcile function. 
+
+Events allow users to see what is going on with a particular object, and allow automated processes to see and respond to them.
+
+ Recent Events for an object can be viewed by running `kubectl describe <resource kind> <resource name>`.
+ 
```

Comment:
```suggestion

 <aside class="warning">
 
Events should be raised in certain circumstances only. Be aware that it is NOT recommended to emit events for all Operations. If authors raise too many events, it brings bad UX experiences for those consuming the solutions on the cluster, and they will not have attention to what they must be notified of. For further information, see the Kubernetes APIs convention describes when using them [here][Events]. 

</aside>
```

It is a very important information. Therefore, we should:
- a) add on the top to ensure that readers will read this one
- b) use the warning class because we are warning users